mkosi/__init__.py

```diff
@@ -934,6 +934,8 @@ def build_initrd(state: MkosiState) -> Path:
         "--make-initrd", "yes",
         "--bootable", "no",
         "--manifest-format", "",
+        *(["--source-date-epoch", state.config.source_date_epoch]
+                if state.config.source_date_epoch else []),
```

Comment:
We should probably do `is not None` here since otherwise we won't pass on a value of zero when building the initrd.